e2e/tests/session_steps_test.go

```diff
@@ -238,7 +238,7 @@ func (s *suite) TheApplicationEstablishesAWebsocketsConnectionForService(appName
 	s.waitForBlockHeight(nextSessionStartHeight)
 
 	// Set WebSocket close height just before the claim window opens
-	s.wsCloseHeight = sharedtypes.GetClaimWindowOpenHeight(&sharedParams, nextSessionStartHeight) - 1
+	s.wsCloseHeight = sharedtypes.GetClaimWindowOpenHeight(&sharedParams, nextSessionStartHeight)
```

Comment:
@red-0ne this is the line I wasn't sure about.

Reply:
If you do a global grep for `.GetClaimWindowOpenHeight(`, it's always one of:
- queryHeight (curr height)
- sessionEndHeight (end of curr session)

It is never `nextSessionStartHeight`.

---

In either case, I'm adding it back and adding a TODO to avoid it blocking this PR.